rpi/default.nix

```diff
@@ -301,9 +312,12 @@ in
         "reset-raspberrypi" # required for vl805 firmware to load
       ];
       # This pin is not necessary, it would be fine to replace it with
-      # `pkgs.rpi-kernels.latest.kernel`. It is helpful to ensure
+      # `kernel`. It is helpful to ensure
       # cache hits for kernel builds though.
-      kernelPackages = pkgs.linuxPackagesFor pkgs.rpi-kernels.latest.kernel;
+      kernelPackages = pkgs.linuxPackagesFor (kernel.override {
+        # Some patches cannot be applied because they are already upstream.
+        ignoreConfigErrors = true;
```

Comment:
Can you motivate this?

Reply:
Latest kernel has patches that could not be overwritten, so rather than not using nixpkgs rpi-kernels nix configuration, I just said, ignore failed patch applies.